website/versioned_docs/version-5.x/api.md

```diff
@@ -42,6 +44,39 @@ Note that if an application is registered from within another application, that
 
 `undefined`
 
+### Configuration object
+```js
+singleSpa.registerApplication({
+	name: 'appName',
+	app: () => System.import('appName'),
+	activeWhen: '/appName'
+	customProps: {}
+})
+```
+
+<h3>arguments</h3>
+
+<dl className="args-list">
+	<dt>name?: string</dt>
+	<dd>App name that single-spa will register and reference this application with, and will be labelled with in dev tools.</dd>
+	<dt>app?: Application | () => Application | Promise&lt;Application&gt; </dt>
+	<dd>Application object or a function that returns the resolved application (Promise or not)</dd>
+	<dt>activeWhen: string | (location) => boolean | (string | (location) => boolean)[]</dt>
+	<dd>Can be a path prefix which will match every URL starting with this path,
```

Comment:
Let's add to this some explanation about dynamic path params in the string. I see you have some detail for it on the other page, but this page should at least mention it.